cmd/pinniped/cmd/oidc_client_options.go

```diff
@@ -19,6 +19,7 @@ import (
 type OIDCClientOptions interface {
 	WithContext(ctx context.Context) oidcclient.Option
 	WithLogger(logger logr.Logger) oidcclient.Option
```

Comment:
We can remove the `WithLogger` option here. Pinniped code should never call this deprecated option.

Reply:
done with next push